group_vars/location_gubener/snmp.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+snmp_devices:
+  - hostname: gubener-sama-5ghz
+    address: 10.31.77.160
```

Comment:
The address is the ip of the device to query. According to `networks.yml` it is the second ip in 10.31.77.160/28.

Mesh internal DNS lookup resolves to .162 

```
spolack@spolack:~$ host gubener-sama.olsr
gubener-sama.olsr has address 10.31.77.162
```

Reply:
Thank you, resolved below.